static/opt-view.js

```diff
@@ -135,22 +125,21 @@ define(function (require) {
                     return x.optType;
                 }
             },"");
+            var contents = _.map(value, this.getDisplayableOpt, this);
             opt.push({
-                range: new monaco.Range(linenumber,1,linenumber,1),
+                range: new monaco.Range(linenumber,1,linenumber,Infinity),
```

Comment:
I use `Infinity` instead of `this.optEditor.getMode()l.getLineMaxColumn(linenumber)` because I did not know I could this. Changing in 3 secs...